utils/include/detray/detectors/build_toy_detector.hpp

```diff
@@ -260,16 +255,45 @@ struct toy_det_config {
     constexpr const auto &barrel_layer_binning() const {
         return m_barrel_binning;
     }
-    constexpr barrel_generator_config<scalar_t> &barrel_config() {
+    constexpr barrel_generator_config<scalar> &barrel_config() {
         return m_barrel_factory_cfg;
     }
-    constexpr endcap_generator_config<scalar_t> &endcap_config() {
+    constexpr endcap_generator_config<scalar> &endcap_config() {
         return m_endcap_factory_cfg;
     }
     constexpr bool do_check() const { return m_do_check; }
     /// @}
 };
 
+/// Print the toy detector configuration
+inline std::ostream &operator<<(std::ostream &out, const toy_det_config &cfg) {
+    out << "\nToy Detector\n"
+        << "----------------------------\n"
+        << "  No. barrel layers     : " << cfg.n_brl_layers() << "\n"
+        << "  No. endcap layers     : " << cfg.n_edc_layers() << "\n"
+        << "  Portal envelope       : " << cfg.envelope() << " [mm]\n";
+
+    if (cfg.use_material_maps()) {
+        /*const auto &cyl_map_bins = cfg.cyl_map_bins();
+        const auto &disc_map_bins = cfg.disc_map_bins();
+
+        out << "  Material maps \n"
+            << "    -> cyl. map bins    : (phi: " << cyl_map_bins[0]
+            << ", z: " << cyl_map_bins[1] << ")\n"
+            << "    -> disc map bins    : (r: " << disc_map_bins[0]
+            << ", phi: " << disc_map_bins[1] << ")\n"
+            << "    -> Min. thickness   : " << cfg.thickness() << " [mm]\n"
+            << "    -> Material         : " << cfg.mapped_material() << "\n";*/
```

Comment:
dead code?

Reply:
Actually code that should be switched on after the latest rebase. Well spotted